cmd/kube-proxy/app/server.go

```diff
@@ -464,20 +438,36 @@ func serveMetrics(bindAddress string, proxyMode kubeproxyconfig.ProxyMode, enabl
 
 	configz.InstallHandler(proxyMux)
 
-	fn := func() {
-		err := http.ListenAndServe(bindAddress, proxyMux)
+	var nodeIPs []net.IP
+	for _, family := range []v1.IPFamily{v1.IPv4Protocol, v1.IPv6Protocol} {
+		npa := proxyutil.NewNodePortAddresses(family, cidrStrings)
+		ips, err := npa.GetNodeIPs(proxyutil.RealNetwork{})
```

Comment:
Using `proxyutil.NodePortAddresses` is wrong here; that has some semantics that don't necessarily apply here, and might have other nodeport-specific behavior in the future.

Possibly some helper should be split out and shared by NodePortAddresses and this code.